Source/PubnubLibrary/Private/FunctionLibraries/PubnubJsonUtilities.cpp

```diff
@@ -6,6 +6,11 @@
 
 FString UPubnubJsonUtilities::JsonObjectToString(TSharedPtr<FJsonObject> JsonObject)
 {
+	if(!JsonObject)
+	{
+		return "";
+	}
+	
```

Comment:
Shouldn't we log anything?

Reply:
This is our inner function. No need to print anything in here.